openmdao/examples/test/test_examples.py

```diff
@@ -4,8 +4,10 @@
    change to the example file.
 """
 
+import inspect
 import sys
 import math
+import pep8
```

Comment:
So you added a dependency on pep8 here but it wasn't added to the setup.py file.  Could we maybe just make this import of pep8 optional and just skip the test if pep8 isn't installed, rather than making it a hard dependency for OpenMDAO or having all of the example tests fail when pep8 isn't installed?